R/matrix.R

```diff
@@ -56,5 +50,13 @@ sleuth_to_matrix <- function(obj, which_df, which_units) {
   s_data$target_id <- NULL
   s_data <- as.matrix(s_data)
 
+  if (obj$gene_mode && which_df == "obs_raw") {
```

Comment:
Maybe you still want to issue a warning here to make sure the user understand the drawbacks of what he's doing? :wink:

Reply:
Sure! I think this is a nice idea to expose the drawbacks.
What do you think that should be warned here?
- `scaled_reads_per_base` is a artificial measure to represent the read counts of genes (in this case, the same warning appear when the user requests `obs_norm`).
- The raw values of `scaled_reads_per_base` are not normalized and, therefore, should not be directly compared.